Incorrect information regarding instant-meilisearch (#1920)
meilisearch/documentation: learn/getting_started/quick_start.md

## Front-end integration

> The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and the open-source [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch generates all the components needed to start searching.

Reviewer: ```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch is the tool that renders all the components needed to start searching.
```

I think I will update the readme of instant meilisearch accordingly :')

> The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, powered by Algolia, is the tool that renders all the components needed to start searching.

Reviewer 2: ```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, an open-source project powered by Algolia, is the tool that renders all the components needed to start searching.
```

> The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and the open-source [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, powered by Algolia, generates all the components needed to start searching.

Author: ```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and the open-source [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch generates all the components needed to start searching.
```

> The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, an open-source project powered by Algolia, is the tool that renders all the components needed to start searching.

Author: ```suggestion
The only step missing now is adding a search bar to your project. The easiest way of achieving this is to use [instant-meilisearch](https://github.com/meilisearch/instant-meilisearch): a plugin that establishes communication between your Meilisearch instance and [InstantSearch](https://github.com/algolia/instantsearch.js). InstantSearch, an open-source project developed by Algolia, is the tool that renders all the components needed to start searching.
```
